features/step_definitions/artifacts.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const { Before, Then, When } = require('cucumber');
+const request = require('screwdriver-request');
+
+const TIMEOUT = 240 * 1000;
+
+Before('@artifacts', function hook() {
+    this.repoName = 'functional-artifacts';
+
+    // Reset shared information
+    this.buildId = null;
+    this.pipelineId = null;
+    this.eventId = null;
+    this.jwt = null;
+});
+
+When(/^the "zipped" job is started$/, { timeout: TIMEOUT }, function step() {
```

Comment:
Perhaps, is it possible to aggregate this function to /^the "main" job is started$/ scenario?
This function looks very similar to the scenario.